core/src/main/java/com/yandex/yoctodb/util/buf/FileChannelBuffer.java

```diff
@@ -279,6 +343,36 @@ protected ByteBuffer initialValue() {
                     return ByteBuffer.allocate(8);
                 }
 
+                @Override
+                public ByteBuffer get() {
+                    final ByteBuffer result = super.get();
+                    result.rewind();
+                    return result;
+                }
+            };
+
+    private final ThreadLocal<ByteBuffer> shortBufCache =
+            new ThreadLocal<ByteBuffer>() {
+                @Override
+                protected ByteBuffer initialValue() {
+                    return ByteBuffer.allocate(Short.BYTES);
```

Comment:
Good! It would be nice to redefine old similar lines above with constants from corresponding java classes instead of magic numbers for code consistency

Reply:
Done